packages/p2p-v2/src/constants/ad-constants.ts

```diff
@@ -20,3 +20,14 @@ export const ADVERT_TYPE = {
     SELL: 'Sell',
     BUY: 'Buy',
 };
+
+export const ERROR_CODES = {
+    AD_EXCEEDS_BALANCE: 'advertiser_balance',
+    AD_EXCEEDS_DAILY_LIMIT: 'advertiser_daily_limit',
+    ADVERT_INACTIVE: 'advert_inactive',
+    ADVERT_MAX_LIMIT: 'advert_max_limit',
+    ADVERT_MIN_LIMIT: 'advert_min_limit',
+    ADVERT_REMAINING: 'advert_remaining',
+    ADVERTISER_ADS_PAUSED: 'advertiser_ads_paused',
+    ADVERTISER_TEMP_BAN: 'advertiser_temp_ban',
+};
```

Comment:
lets move these constants into a different constant file for api-error-codes, similar to the one we have in p2p-v1 and also wrap the object with `Object.freeze`